build.gradle

```diff
@@ -130,16 +130,20 @@ repositories {
         name = "MelanX Maven"
         url = "https://maven.melanx.de/"
     }
+    maven {
+        url = "https://www.cursemaven.com"
+    }
 }
 
 dependencies {
     minecraft "net.minecraftforge:forge:${mc_version}-${forge_version}"
-    runtimeOnly fg.deobf("mezz.jei:jei-1.16.2:${jei_version}")
+    compile fg.deobf("mezz.jei:jei-1.16.2:${jei_version}")
 
     compile fg.deobf("vazkii.botania:Botania:${botania_version}")
     runtime fg.deobf("top.theillusivec4.curios:curios-forge:${mc_version}-${curios_version}")
     compile fg.deobf("vazkii.patchouli:Patchouli:${patchouli_version}")
     compile fg.deobf("io.github.noeppi_noeppi.mods:LibX:${mc_version}-${libx_version}")
+    runtime fg.deobf("curse.maven:create-328085:3167531")
```

Comment:
I think this should use `runtimeOnly`

Reply:
It was there only for testing purposes, i killed it.